docs/how-to/package-with-hatch.md

```diff
@@ -0,0 +1,380 @@
+# Package a Textual app with Hatch
+
+Python apps may be distributed via [PyPi](https://pypi.org/) so they can be installed via `pip`.
```

Comment:
```suggestion
Python apps may be distributed via [PyPI](https://pypi.org/) so they can be installed via `pip`.
```

Reply:
PYthon Packing Index
Might be worth a search + replace for this typo. It shows up a couple of times and I'm not sure I marked all instances.